CEXT-1818: Commerce-eventing module events subscriptions (#126)
AdobeDocs/commerce-events: src/pages/get-started/configure-commerce.md

   ```bash
   bin/magento events:subscribe observer.customer_login --fields=firstname --fields=lastname --fields=entity_id
   ```
   

>    **Warning**: If you subscribe to events with `events:subscribe` command the configuration will be stored in `app/etc/config.php` and you should keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment so as result the event subscription will be lost as well.

Reviewer: @oshmyheliuk Does this warning apply to the event provider configuration for a Commerce app deployed on cloud as well?

Author: No, the event provider configuration is not stored in `app/etc/config.php`, it's stored in the database, so it won't be accidentally erased during redeployment.
This warning is only related to event subscriptions via the CLI command.

>    **Warning**: If you subscribe to events with `events:subscribe` command the configuration will be stored in `app/etc/config.php` and you should keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment so as result the event subscription will be lost as well.

Reviewer 2: ```suggestion
   **Warning**: When you use the `events:subscribe` command to subscribe to events on an on-premises installation, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well. Cloud deployments do not store this information in the `config.php`, and event subscriptions are not affected when Commerce is redeployed.
```

>    **Warning**: When you use the `events:subscribe` command to subscribe to events on a Cloud environment, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well.

Reviewer 2: ```suggestion
   When you use the `events:subscribe` command to subscribe to events on a Cloud environment, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well.
```

>    To make these changes permanent please add appropriate configuration to the `app/etc/config.php` file under Git.

Reviewer 2: ```suggestion
   To make these changes permanent, manually add the appropriate configuration to the `app/etc/config.php` file under Git.
```
